tests/Dockerfile

```diff
@@ -47,16 +45,20 @@ RUN dnf -y update && \
         tar \
         virt-install \
         zanata-client && \
-        dnf -y builddep /tmp/cockpit.spec && dnf clean all
-RUN npm -g install phantomjs-prebuilt
+    npm -g install phantomjs-prebuilt
+
+# conserving layers, that's why we are not using ADD
+RUN curl -s -o /tmp/cockpit.spec https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/cockpit.spec && \
+    dnf -y builddep /tmp/cockpit.spec && \
+    dnf clean all
```

Comment:
This is actually a case where you do need to merge the `RUN` commands. Running `dnf clean all` in a separate one makes it useless, as all the yum caching is already in the previous image layer. That's why I previously rearranged it like that. Calling curl in the middle instead of the `ADD` is fine too, though.